### PR TITLE
Update vocab.csv

### PR DESCRIPTION
**Pull request by sauuyer**
**Updated with METBKA details according to CGSN Trello card: METBK Refdes Vocab Issues [DT-244]**

- https://trello.com/c/ZuakyBkp/244-metbk-refdes-vocab-issues-dt-244?search_id=7b2b3d4d-6849-48f1-b99f-e5c9e86f0159

Reference files used:
- [METBK_Metadata_Updates_2024-05-01.xlsx](https://github.com/oceanobservatories/asset-management/files/15422555/METBK_Metadata_Updates_2024-05-01.xlsx)
- [METBK_updated_values.csv](https://github.com/oceanobservatories/asset-management/files/15422556/METBK_updated_values.csv)

### CODE REVIEW

File: vocab/vocab.csv [hunk 1]
```diff
@@ -88,14 +88,14 @@ CE02SHSM-SBD11-00-DCLENG000,Coastal Endurance,Oregon Shelf Surface Mooring,Surfa
...
+CE02SHSM-SBD11-06-METBKA002,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CE02SHSM-SBD11-06-METBKA003,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CE02SHSM-SBD11-06-METBKA004,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CE02SHSM-SBD11-06-METBKA005,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CE02SHSM-SBD11-06-METBKA006,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CE02SHSM-SBD11-06-METBKA007,Coastal Endurance,Oregon Shelf Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:98]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 2]
```diff
@@ -502,14 +502,14 @@ CE09OSSM-SBD11-00-DCLENG000,Coastal Endurance,Washington Offshore Surface Moorin
...
+CE09OSSM-SBD11-06-METBKA002,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CE09OSSM-SBD11-06-METBKA003,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CE09OSSM-SBD11-06-METBKA004,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CE09OSSM-SBD11-06-METBKA005,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CE09OSSM-SBD11-06-METBKA006,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CE09OSSM-SBD11-06-METBKA007,Coastal Endurance,Washington Offshore Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:512]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 3]
```diff
@@ -564,27 +564,27 @@ CP01CNSM-SBD11,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,,,,0,0
...
+CP01CNSM-SBD12-06-METBKA001,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+CP01CNSM-SBD12-06-METBKA002,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CP01CNSM-SBD12-06-METBKA003,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CP01CNSM-SBD12-06-METBKA004,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CP01CNSM-SBD12-06-METBKA005,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CP01CNSM-SBD12-06-METBKA006,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CP01CNSM-SBD12-06-METBKA007,Coastal Pioneer NES,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:587]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 4]
```diff
@@ -965,27 +965,27 @@ CP10CNSM-SBD11-00-DCLENG000,Coastal Pioneer MAB,Central Surface Mooring,Surface
...
+CP10CNSM-SBD11-06-METBKA001,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+CP10CNSM-SBD11-06-METBKA002,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CP10CNSM-SBD11-06-METBKA003,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CP10CNSM-SBD11-06-METBKA004,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CP10CNSM-SBD11-06-METBKA005,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CP10CNSM-SBD11-06-METBKA006,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CP10CNSM-SBD11-06-METBKA007,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:975]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 5]
```diff
@@ -965,27 +965,27 @@ CP10CNSM-SBD11-00-DCLENG000,Coastal Pioneer MAB,Central Surface Mooring,Surface
...
+CP10CNSM-SBD12-06-METBKA001,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+CP10CNSM-SBD12-06-METBKA002,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CP10CNSM-SBD12-06-METBKA003,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CP10CNSM-SBD12-06-METBKA004,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CP10CNSM-SBD12-06-METBKA005,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CP10CNSM-SBD12-06-METBKA006,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CP10CNSM-SBD12-06-METBKA007,Coastal Pioneer MAB,Central Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:988]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 6]
```diff
@@ -1028,14 +1028,14 @@ CP11NOSM-SBD11-00-DCLENG000,Coastal Pioneer MAB,Northern Surface Mooring,Surface
...
+CP11NOSM-SBD11-06-METBKA001,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+CP11NOSM-SBD11-06-METBKA002,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CP11NOSM-SBD11-06-METBKA003,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CP11NOSM-SBD11-06-METBKA004,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CP11NOSM-SBD11-06-METBKA005,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CP11NOSM-SBD11-06-METBKA006,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CP11NOSM-SBD11-06-METBKA007,Coastal Pioneer MAB,Northern Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1038]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 7]
```diff
@@ -1082,14 +1082,14 @@ CP11SOSM-SBD11-00-DCLENG000,Coastal Pioneer MAB,Southern Surface Mooring,Surface
...
+CP11SOSM-SBD11-06-METBKA001,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+CP11SOSM-SBD11-06-METBKA002,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-4,-4
+CP11SOSM-SBD11-06-METBKA003,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-4,-4
+CP11SOSM-SBD11-06-METBKA004,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-4,-4
+CP11SOSM-SBD11-06-METBKA005,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-4,-4
+CP11SOSM-SBD11-06-METBKA006,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-4,-4
+CP11SOSM-SBD11-06-METBKA007,Coastal Pioneer MAB,Southern Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-5,-5
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1092]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 8]
```diff
@@ -1302,14 +1302,14 @@ GA01SUMO-SBD11-01-MOPAK0000,Global Argentine Basin,Apex Surface Mooring,Surface
...
+GA01SUMO-SBD11-06-METBKA001,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GA01SUMO-SBD11-06-METBKA002,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GA01SUMO-SBD11-06-METBKA003,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GA01SUMO-SBD11-06-METBKA004,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GA01SUMO-SBD11-06-METBKA005,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GA01SUMO-SBD11-06-METBKA006,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GA01SUMO-SBD11-06-METBKA007,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1312]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 9]
```diff
@@ -1318,14 +1318,14 @@ GA01SUMO-SBD12-02-FLORTD000,Global Argentine Basin,Apex Surface Mooring,Surface
...
+GA01SUMO-SBD12-06-METBKA001,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GA01SUMO-SBD12-06-METBKA002,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GA01SUMO-SBD12-06-METBKA003,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GA01SUMO-SBD12-06-METBKA004,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GA01SUMO-SBD12-06-METBKA005,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GA01SUMO-SBD12-06-METBKA006,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GA01SUMO-SBD12-06-METBKA007,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1328]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 10]
```diff
@@ -1502,14 +1502,14 @@ GI01SUMO-SBD11-01-MOPAK0000,Global Irminger Sea,Apex Surface Mooring,Surface Buo
...
+GI01SUMO-SBD11-06-METBKA001,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GI01SUMO-SBD11-06-METBKA002,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GI01SUMO-SBD11-06-METBKA003,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GI01SUMO-SBD11-06-METBKA004,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GI01SUMO-SBD11-06-METBKA005,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GI01SUMO-SBD11-06-METBKA006,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GI01SUMO-SBD11-06-METBKA007,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1512]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 11]
```diff
@@ -1518,14 +1518,14 @@ GI01SUMO-SBD12-02-FLORTD000,Global Irminger Sea,Apex Surface Mooring,Surface Buo
...
+GI01SUMO-SBD12-06-METBKA001,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GI01SUMO-SBD12-06-METBKA002,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GI01SUMO-SBD12-06-METBKA003,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GI01SUMO-SBD12-06-METBKA004,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GI01SUMO-SBD12-06-METBKA005,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GI01SUMO-SBD12-06-METBKA006,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GI01SUMO-SBD12-06-METBKA007,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1528]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 12]
```diff
@@ -1957,14 +1957,14 @@ GS01SUMO-SBD11-03-HYDGN0000,Global Southern Ocean,Apex Surface Mooring,Surface B
...
+GS01SUMO-SBD11-06-METBKA001,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GS01SUMO-SBD11-06-METBKA002,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GS01SUMO-SBD11-06-METBKA003,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GS01SUMO-SBD11-06-METBKA004,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GS01SUMO-SBD11-06-METBKA005,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GS01SUMO-SBD11-06-METBKA006,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GS01SUMO-SBD11-06-METBKA007,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1967]: looks good - matches suggested edits per SW/CW

File: vocab/vocab.csv [hunk 13]
```diff
@@ -1973,14 +1973,14 @@ GS01SUMO-SBD12-02-FLORTD000,Global Southern Ocean,Apex Surface Mooring,Surface B
...
+GS01SUMO-SBD12-06-METBKA001,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Surface CT,Sea-Bird Electronics,SBE 37-SM MicroCAT CT Recorder,1,1
+GS01SUMO-SBD12-06-METBKA002,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology BPR,Star Engineering,ASIMET Barometric Pressure Module (BPR),-5,-5
+GS01SUMO-SBD12-06-METBKA003,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology HRH,Star Engineering,ASIMET Relative Humidity & Air Temperature Module (HRH),-5,-5
+GS01SUMO-SBD12-06-METBKA004,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology LWR,Star Engineering,ASIMET Longwave Radiation Module (LWR),-5,-5
+GS01SUMO-SBD12-06-METBKA005,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology PRC,Star Engineering,ASIMET Precipitation Module (PRC),-5,-5
+GS01SUMO-SBD12-06-METBKA006,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWR,Star Engineering,ASIMET Shortwave Radiation Module (SWR),-5,-5
+GS01SUMO-SBD12-06-METBKA007,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology SWND,Star Engineering,ASIMET Sonic Wind Module (SWND),-6,-6
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1983]: looks good - matches suggested edits per SW/CW

**Review by rtravis-whoi — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[METBK Refdes Vocab Issues [DT-244]](https://trello.com/c/ZuakyBkp/244-metbk-refdes-vocab-issues-dt-244)

**Review by cwingard — APPROVED** [review]
Looks good, thanks for taking care of this!

**Pull request merged**